templates/partials/scheduled_maintenance.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+
+{% load staticfiles %}
+
+{% if is_maintenance_scheduled and maintenance_start_datetime and maintenance_end_datetime%}
+<div class="alert alert-warning" role="alert">
+  There will be future maintenance on {{ maintenance_start_datetime }} until {{ maintenance_end_datetime }}
```

Comment:
You should change this to something like 

> Please note, there will be scheduled downtime on {{ maintenance_start_datetime }} until {{ maintenance_end_datetime }}. We will be using this time to update our infrastructure.
> 
> **Any graph updated or uploaded during the maintenance period will be lost. Please plan your work accordingly.**
> 
> We appreciate your patience and apologize for any inconvenience.

Reply:
I changed the warning message to look like this:
![image](https://user-images.githubusercontent.com/35696537/104221912-966e8480-540f-11eb-909e-8eba4ba57eae.png)